storage/src/vespa/storage/common/storagelink.h

```diff
@@ -199,7 +201,7 @@ class Queue {
      * (0 = don't wait, -1 = forever)
      * @return  true or false if the queue was empty.
      */
-    bool getNext(std::shared_ptr<api::StorageMessage>& msg, int timeout);
+    bool getNext(std::shared_ptr<api::StorageMessage>& msg, vespalib::duration timeout);
```

Comment:
Consider removing comment about "-1 = waiting forever", as that probably doesn't work anymore (did it ever?)

Reply:
Done.